scripts/clog.inputs

```diff
@@ -1,4 +1,4 @@
-../src/platform/selfsign_openssl.c
+../src/platform/selfsign_quictls.c
```

Comment:
Similar to crypt, I think we should keep the name as is, because quictls doesn't fork this.

Reply:
This still isn't fixed.